src/Directory.build.props

```diff
@@ -34,4 +30,26 @@
         <!-- https://github.com/dotnet/sourcelink/blob/master/docs/README.md#embedallsources -->
         <EmbedAllSources>true</EmbedAllSources>
     </PropertyGroup>
+    <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
```

Comment:
Aren't the blocks below exactly the same for each framework?

Reply:
Yes, reducing them... in an initial (pre-commit) state there was something else.